po/radxa-welcome.pot

```diff
@@ -8,20 +8,20 @@ msgid ""
 msgstr ""
 "Project-Id-Version: radxa-welcome\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2024-07-03 11:19+0300\n"
+"POT-Creation-Date: 2024-07-22 04:08+0300\n"
 "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
```

Comment:
More placeholder values.

Reply:
Those are actually meant to be placeholder values

Reply:
Since the pot isn't a translation for any language it has placeholders take a look at zh_cn.po it autofilled there